tests/integration_tests/validation/conformance_suite_configurations/xbrl_report_packages_1_0.py

```diff
@@ -0,0 +1,112 @@
+from pathlib import PurePath, Path
+from tests.integration_tests.validation.conformance_suite_config import ConformanceSuiteConfig, ConformanceSuiteAssetConfig
+
+config = ConformanceSuiteConfig(
+    assets=[
+        ConformanceSuiteAssetConfig.conformance_suite(
+            Path("report-package-conformance.zip"),
+            entry_point=Path("report-package-conformance/index.csv"),
+        ),
+    ],
+    expected_failure_ids=frozenset(f"report-package-conformance/index.csv:{s}" for s in [
```

Comment:
This list seems to have ~all the tests as expected failure, which seems surprising to me since Arelle does have some support for report packages.

Perhaps related, or perhaps I'm just doing it wrong, but when I run locally, it only tries to run one test and fails as if it's trying to read index.csv as a zip file.
```
$ python -m tests.integration_tests.validation.run_conformance_suites --name xbrl_report_packages_1_0 --test --offline > log.txt
[info] loaded in 0.01 secs at 2024-07-01T15:10:10 - /git/arelle/tests/resources/conformance_suites/report-package-conformance.zip/report-package-conformance/index.csv
[info] Testcase - index.csv
[info] Variation V-000-invalid-zip V-000-invalid-zip: ['rpe:invalidArchiveFormat', 'tpe:invalidArchiveFormat'] - A report package MUST conform to the .ZIP File Format Specification - index.csv
[exception:BadZipFile] Testcase validation exception: File is not a zip file, testcase: index.csv - index.csv
Traceback (most recent call last):
  File "/git/arelle/arelle/Validate.py", line 102, in validate
    self.validateTestcase(self.modelXbrl.modelDocument)
  File "/git/arelle/arelle/Validate.py", line 275, in validateTestcase
    filesource = FileSource.openFileSource(readMeFirstUri, self.modelXbrl.modelManager.cntlr, base=baseForElement,
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/git/arelle/arelle/FileSource.py", line 71, in openFileSource
    filesource.open(reloadCache)
  File "/git/arelle/arelle/FileSource.py", line 203, in open
    self.fs = zipfile.ZipFile(fileStream, mode="r")
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.9/Frameworks/Python.framework/Versions/3.11/lib/python3.11/zipfile.py", line 1312, in __init__
    self._RealGetContents()
  File "/opt/homebrew/Cellar/python@3.11/3.11.9/Frameworks/Python.framework/Versions/3.11/lib/python3.11/zipfile.py", line 1379, in _RealGetContents
    raise BadZipFile("File is not a zip file")
zipfile.BadZipFile: File is not a zip file
[info] validated in 0.00 secs - /git/arelle/tests/resources/conformance_suites/report-package-conformance.zip/report-package-conformance/index.csv
```

Reply:
The list is heavy on the error cases because the suite is looking for specific report package error codes, and Arelle doesn't emit any of those right now. The issue with it seeming like it's trying to read the `index.csv` as a zip file is actually a bit of a mix-up. It's not about the csv; it's stumbling over the `V-000-invalid-zip` file, which it can't open as a zip. 

We've got a few ways to run these conformance suite tests in Arelle, and my initial setup treated that exception as an expected failure. But after opening this PR I realized that's not the case across all of the ways we can run conformance suite tests or the CI setup. I've tweaked things to fix that. Should be all good now, but let me know if anything else pops up or if you've got more feedback.

Reply:
> It's not about the csv; it's stumbling over the V-000-invalid-zip file, which it can't open as a zip.

💡 👍 

> Should be all good now, but let me know if anything else pops up or if you've got more feedback.

The test result output looks like a better starting point now.